package.json

```diff
@@ -197,7 +197,7 @@
       "!packages/create-keystone-app/templates/**",
       "!packages/core/tests/default-entry/index.js",
       "!packages/fields/**/views/**",
-      "!packages/fields/**/filterTests.js",
+      "!packages/fields/**/test-fixtures.js",
```

Comment:
Why the file renames? In adding the `crudTests` I realised the filename was misleading, so picked one I thought was general enough while still conveying the overall meaning.

Happy to be challenged on this one!